Jenkins/jenkins.yaml

```diff
@@ -0,0 +1,236 @@
+jenkins:
+  agentProtocols:
+  - "JNLP4-connect"
+  - "Ping"
+  authorizationStrategy: "loggedInUsersCanDoAnything"
+  crumbIssuer:
+    standard:
+      excludeClientIPFromCrumb: true
+  disableRememberMe: false
+  globalNodeProperties:
+  - envVars:
+      env:
+      - key: "ContainerRegistryCredId"
+        value: "ecr:us-east-1:mehrab-trg00000835-opsusr"
+      - key: "ContainerRegistryURL"
+        value: "https://855430746673.dkr.ecr.us-east-1.amazonaws.com"
+      - key: "kubectlCreds"
+        value: "kubectl-token"
+      - key: "kubectlServer"
+        value: "https://kubernetes.default.svc"
```

Comment:
Add helmChartRepoURL to env if there's time: http://a9c2594943a63472faa675f1858e4264-18446664.us-east-1.elb.amazonaws.com/api/charts